src/client/script.js

```diff
@@ -63,36 +64,36 @@ function isValidSpace(spaceId) {
 function Board() {
 	this.spaces = {};
 
-	for (var i=1; i<=8; i++) {
-		for (var j=1; j<=8; j++) {
-			var xPos = intCoordsToSpaceId(i, j)[0]; 
-			var yPos = intCoordsToSpaceId(i, j)[1]; 
+	for (var i = 1; i <= 8; i++) {
```

Comment:
minor formatting changes, like adding whitespace around operators, are fine but they mainly just pollute the diff and make it harder to merge in your changes

Reply:
but i'll resolve that myself on merging